app/views/application/index.html.haml

```diff
@@ -1,10 +1,10 @@
 !!!
-%html{'ng-app' => 'loomioApp', 'ng-controller' => 'RootController as app'}
+%html{'ng-app' => 'loomioApp', 'ng-controller' => 'RootController as app', 'lang' => I18n.locale}
   %head
     %title= AppConfig.theme[:site_name]
     %meta{charset: 'utf-8'}
     %meta{content: 'IE=Edge', 'http-equiv' => 'X-UA-Compatible'}
-    %meta{content: 'width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no', name: "viewport"}
+    %meta{content: 'width=device-width, initial-scale=1, maximum-scale=5', name: "viewport"}
```

Comment:
is this recommendly a11y practice?

Reply:
This was picked up by Lighthouse, with this resource to learn more: https://dequeuniversity.com/rules/axe/2.2/meta-viewport?application=lighthouse